CleanPy/replace_na.py

```diff
@@ -1,34 +1,60 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Nov 22 10:02:49 2018
 
-def replace_na(data, columns, replace="average", remove=False):
+@author: phuntsoktseten
+"""
+import pandas as pd
+import numpy as np 
+
+def replace_na(data, columns, replace="mean", remove=False):
     """
-    This function replaces NA values with either the min, max, median or average 
+    This function replaces NA values with either the min, max, median or mean
     value or removes the rows.
-
     Parameters
     ----------
     data : dataframe
         This is the dataframe that the function will use to replace NAs.
         
+    columns : list
+        List of columns to replace missing values on.
```

Comment:
should this be list of column indices to replace missing values on? We should modify the code too so that it's easier for the user to specify which column(s) they want to replace